docs/mcr-endpoints-guidance.md

```diff
@@ -36,6 +42,8 @@ MAR registry REST endpoints are available in the following nine locations:
 - Asia South East
 - Europe North
 - Europe West
+- North China
+- East China
```

Comment:
```suggestion
- China North 3
- China East 3
```
See naming in below links:
- https://www.azurespeed.com/Information/AzureRegions/ChinaNorth3
- https://www.azurespeed.com/Information/AzureRegions/ChinaEast3